packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -73,17 +73,22 @@ const generateHeaderCellForPrimitives = toolbarProps => {
     </TableCell>
   );
 };
+const getValidColumnProps = (scopedSchema: JsonSchema) =>
```

Comment:
This seems to crash when first clicking 'select all', followed by a delete. We should also add a test case covering this scenario.